### PR TITLE
feat(radio): refresh radio UI

### PR DESCRIPTION
**Pull request by MNZhu**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: examples/sites/demos/mobile/app/radio/webdoc/radio.js [hunk 1]
```diff
@@ -84,7 +50,7 @@ export default {
             'zh-CN': '<p>Radio 的 value,该属性默认为{}</p>',
             'en-US': 'display different button'
           },
-          demoId: 'event-change'
+          demoId: 'base'
```
Comment (gimmyhehe) [inline examples/sites/demos/mobile/app/radio/webdoc/radio.js]: 这里的demo和Switch一样，拆分一下特性demo较好~

File: examples/sites/demos/mobile/app/radio/base.vue [hunk 2]
```diff
@@ -1,95 +1,33 @@
...
-        :content-des="item.contentdes"
-        @click="clickFn"
-      >
-        <template #suffix>
-          <tiny-radio v-model="value1" :label="item.id"></tiny-radio>
-        </template>
-      </tiny-list>
-    </div>
+    <p>默认</p>
+    <tiny-radio v-model="value" label="1" @change="changeAction">男</tiny-radio>
```
Comment (gimmyhehe) [inline examples/sites/demos/mobile/app/radio/base.vue]: Radio组件建议和Switch一样，拆分特性demo。事件合并成一个demo切放在最后。

File: examples/sites/demos/mobile/app/radio/base.vue [hunk 3]
```diff
@@ -1,95 +1,33 @@
...
+    <tiny-radio v-model="value" label="2" @change="changeAction">女</tiny-radio>
   </div>
 </template>
 
 <script lang="jsx">
```
Comment (gimmyhehe) [inline examples/sites/demos/mobile/app/radio/base.vue]: 虽然这是之前遗留的问题，这里可以顺手帮忙删除一下~。如果没有用到jsx语法的话，建议删除一下lang="jsx"~

File: examples/sites/demos/mobile/app/radio/base.vue [hunk 4]
```diff
@@ -1,95 +1,33 @@
...
+      value: '1'
     }
   },
   methods: {
-    clickFn(list) {
-      if (list.id) {
-        this.value1 = list.id
-      }
+    changeAction(value) {
+      console.log(value)
```
Comment (gimmyhehe) [inline examples/sites/demos/mobile/app/radio/base.vue]: 事件触发提示不建议用console.log，因为用户比较难感知到，我们其他demo的做法是使用Modal.message弹出一个消息。

File: examples/sites/demos/mobile/app/radio/disabled.vue [hunk 5]
```diff
@@ -16,12 +17,46 @@ export default {
     return {
       value: '1'
     }
+  },
+  methods: {
```
Comment (gimmyhehe) [inline examples/sites/demos/mobile/app/radio/disabled.vue]: 建议删除无用代码，禁用特性的demo，看起来没有使用到此方法。

File: examples/sites/demos/mobile/app/radio/base.vue (hunk 4 above)
Comment (MNZhu) [reply examples/sites/demos/mobile/app/radio/base.vue]: 已换成页面展示

File: examples/sites/demos/mobile/app/radio/disabled.vue (hunk 5 above)
Comment (MNZhu) [reply examples/sites/demos/mobile/app/radio/disabled.vue]: 已删除

**Review by zzcr — APPROVED** [review]

**Pull request merged**